src/EditorFeatures/Core.Wpf/WpfClassificationExtensions.cs

```diff
@@ -66,15 +66,22 @@ public static Run ToRun(this ClassifiedText part, IClassificationFormatMap forma
             ClassificationTypeMap typeMap)
         {
             var inlines = parts.ToInlines(formatMap, typeMap);
-            return inlines.ToTextBlock(formatMap, typeMap);
+            return inlines.ToTextBlock(formatMap);
         }
 
+        [Obsolete("Use 'public static TextBlock ToTextBlock(this IEnumerable <Inline> inlines, IClassificationFormatMap formatMap, bool wrap = true)' instead")]
         public static TextBlock ToTextBlock(
             this IEnumerable<Inline> inlines,
             IClassificationFormatMap formatMap,
             ClassificationTypeMap typeMap,
             string classificationFormatMap = null,
             bool wrap = true)
+            => inlines.ToTextBlock(formatMap, wrap);
```

Comment:
why do we need this helper?

Reply:
We keep the old one to avoid breaking partner teams. We are informing them that the method is obsolete and then will remove the method soon.

The new method is introduced to remove unnecessary parameters.

---
In reply to: [247302300](https://github.com/dotnet/roslyn/pull/32307#discussion_r247302300) [](ancestors = 247302300)